internal/e2e/tester/cardinal/sys/join.go

```diff
@@ -26,6 +28,7 @@ func Join(ctx cardinal.WorldContext) error {
 		}
 		PlayerEntityID[jtx.Sig().PersonaTag] = entityID
 		logger.Info().Msgf("player %s successfully joined", jtx.Sig().PersonaTag)
+		ctx.EmitEvent(fmt.Sprintf("%d player: %d created, %d/%d", entityID, entityID, entityID, 3))
```

Comment:
- not sure we need the "3" at the end
- do we need to list the entity id 3 times?

Reply:
hmm this is for a test. Previously I could actually get the amount transactions. Now with this new method there's no access to the length. I was thinking that in the future I could create a method that could access the length and replace it here but I think I'm just going to modify the test.

Reply:
saw the fix, but still seems unnecessary to send the entity ID 3 times here, we can just simplify to 1?

Reply:
Well, this test actually caught one of the huge bugs I was trying to work on. Basically events were lossy, if I sent events too fast I would actually be missing events. If I didn't send 3 this bug would not have been caught.

Reply:
missing event data or missing entire events? my point is that you have 
```go
ctx.EmitEvent(fmt.Sprintf("%d player: %d created, %d/%d", entityID, entityID, entityID, 3))
```
when you could just do 
```go
ctx.EmitEvent(fmt.Sprintf("player entity id %d created", entityID))
```

Reply:
I meant Missing entire events. 

Ok I thought you meant emitting the id, three times not the string itself. 

Yeah, idc, The test, however requires the first value to be the entityid. So I'll just make it work with that.